### PR TITLE
feat(tests): add unit tests for unit converter

### PR DESCRIPTION
**Pull request by juniormp**
This PR contains:

Added unit test for `unit-converter` class.

Tested components:
- `numberToString`
- `scaleDecimals`
- `UNIT_CONVERTERS`

Notes: To avoid conflicts with other test files I'm using `simon` sandbox feature to isolate environments.

### CODE REVIEW

File: src/test/unit-converter.test.ts [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+      const decimals = 10;
+      const weiValue = scaleDecimals("123.400000", decimals);
+      chai.expect(weiValue).to.equal("1234000000000");
+    });
+
+    it("Should throw an error for too many decimal points", async () => {
+      const decimals = 18;
+      chai
+        .expect(() => scaleDecimals("1.2.3", decimals))
+        .to.throw(Error, "[ethjs-unit] while converting number 1.2.3 to wei,  too many decimal points");
```
Comment (tcoratger) [inline src/test/unit-converter.test.ts:157]: @juniormp Small detail: seems like there is an additional useless space `to wei,  too many decimal points` should become `to wei, too many decimal points`, can you correct that in the unit tests as well as in the `unitConverter.ts` file? This is the good PR to do that :)

File: src/test/unit-converter.test.ts [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+          "while converting number to string, invalid number value '[object Object]' type object."
+        );
+      chai
+        .expect(() => numberToString([]))
+        .to.throw(Error)
+        .with.property("message", "while converting number to string, invalid number value '' type object.");
+    });
+  });
+
+  describe("scaleDecimals", async function () {
```
Comment (tcoratger) [inline src/test/unit-converter.test.ts:122]: @juniormp The following case seems to be not covered here:

https://github.com/grindery-io/grindery-nexus-connector-web3/blob/907e72a3615ddf6326b63635199d835a2dc43c89/src/web3/evm/unitConverter.ts#L38

Can you add it?

**Comment by tcoratger** [comment]
@juniormp Thanks, looks good to me, pending @SAPikachu :)

**Review by SAPikachu — APPROVED** [review]
Looks good to me, thanks!

**Pull request merged**